tox.ini

```diff
@@ -19,4 +19,4 @@ commands =
 basepython = python2
 skip_install = true
 deps = flake8
-commands = flake8 --ignore E731 --exclude .tox,.git
+commands = flake8 --ignore E731,W504,W503 --exclude .tox,.git
```

Comment:
W504 is according to an actual PEP8 standard, while W503 is obsolete. You have to choose which standard you want to follow because they are contrary.